pkg/templates/notificationtemplates/notification_generator.go

```diff
@@ -9,17 +9,18 @@ import (
 )
 
 var notificationTemplates map[string]NotificationTemplate
+var UserProvisioned, _, _ = GetNotificationTemplate("userprovisioned")
```

Comment:
Make sure we have a test to check that UserProvisioned is not an empty struct and actually equals the expected template (with "userprovisioned" name).

Reply:
done here: c523c03158f3fd42722c96bd2446973d15a3a2bd